libs/novui/src/json-schema-components/JsonSchemaForm.tsx

```diff
@@ -76,6 +107,11 @@ export function JsonSchemaForm<TFormData = any>(props: JsonSchemaFormProps<TForm
       uiSchema={UI_SCHEMA}
       widgets={isAutocompleteEnabled ? WIDGETS : LEGACY_WIDGETS}
       validator={validator}
+      // override default behavior don't print to console when there is a validation error
+      onError={() => {}}
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
👏 **praise:** Nice catch 👏‏